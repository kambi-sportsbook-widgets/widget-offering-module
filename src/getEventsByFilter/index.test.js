import { getEventsByFilter } from './index'

jest.setTimeout(30000) // need to increase default timeout for functions as we're making network requests

describe('Test suite for getEventsByFilter(filter) offering module function', () => {
  it(`it gets FOOTBALL events from kambi offering API`, () => {
    return getEventsByFilter('/football').then(data => {
      // expect(data).toHaveProperty('event')
      console.log(data)
    })
  })
})
