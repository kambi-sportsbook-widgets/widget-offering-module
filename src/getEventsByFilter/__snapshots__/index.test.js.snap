// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suite for getEventsByFilter(filter) offering module function it tries to get filters with a number 123 1`] = `"A string filter must be defined to query against the Kambi Offering API. E.g. \\"/football\\""`;
